src/applications/check-in/day-of/tests/e2e/translation/validate-veteran-page-is-translated.cypress.spec.js

```diff
@@ -46,12 +46,11 @@ describe('Check In Experience - Tagalog', () => {
   it('Validate Veteran page - tagalog', () => {
     // App is translated.
     ValidateVeteran.validatePage.dayOf('tl');
-    // NOTE: DS components don't seem to support tagalog..
     cy.get('[data-testid="last-name-input"]')
       .shadow()
       .find('label')
       .should('be.visible')
-      .and('have.text', 'Ang iyong apelyido (*Required)');
+      .and('have.text', 'Ang iyong apelyido (*Kailangan)');
```

Comment:
@patient-check-in This PR has updated the component library, so this test needed updating to pass

Reply:
Thanks for the update!